src/scheduling.py

```diff
@@ -42,24 +43,30 @@ def logout():
 
     @app.route("/login", methods=["POST"])
     def login():
+        """Using POST method for Employee login"""
         email = request.form["email"]
         password = request.form["password"]
         user = database.find_user(email)
+
         if user is None and not database.get_users():
+            # Creates Employee in the database if not exists
             user = database.create_user(
                 email, password, "Admin", hours_limit=0.0, admin=True
             )
         elif user is None:
+            """Redirects the user if not found in the database"""
```

Comment:
This can be a regular comment. doc-strings should only be right after `def` and `class` statements.

Reply:
Ok I got it.